script.R

```diff
@@ -61,25 +117,11 @@ cli::cli_bullets(refs)
 cli::cli_h1("Initiate pre-requisites")
 cli::cli_progress_bar()
 
-## install pkg
-# cache pkg and its dependencies
-cli::cli_progress_step("Installing the package...")
-pkg_name <- read.dcf("DESCRIPTION")[1, "Package"][[1]]
-crancache::install_packages(pkg_name, quiet = TRUE)
-
-## revdepcheck
-cli::cli_progress_step("Initiating `revdepcheck`...")
-revdepcheck::revdep_reset()
-unlink("./revdep/", recursive = TRUE)
-revdepcheck:::db_disconnect(".")
-usethis::use_revdep()
-revdepcheck:::db_setup(".")
-
 ## miniCRAN
 cli::cli_progress_step("Initiating `miniCRAN`...")
 minicran_path <- tempfile()
 dir.create(minicran_path)
-# added `rlang` as a dummy package as the `pkgs` arg cannot be empty
```

Comment:
I would not delete that comment as it is useful to understand why this package was used in the next `miniCRAN::makeRepo` call

Reply:
ok